resources/definitions/ultimaker_method_base.def.json

```diff
@@ -390,8 +392,9 @@
         "speed_wall_0": { "value": "speed_wall * 30/40" },
         "speed_wall_x": { "value": "speed_wall" },
         "support_angle": { "value": 40 },
-        "support_bottom_distance": { "value": "layer_height if extruder_nr == support_extruder_nr else 0" },
+        "support_bottom_distance": { "value": "layer_height if extruder_nr == support_extruder_nr else support_z_distance" },
         "support_bottom_enable": { "value": "false if extruder_nr == support_extruder_nr else true" },
```

Comment:
This is actually incorrect: when using a formula, it is interpreted as Python code, so `false` should be `False` and `true` should be `True`

Reply:
I published a new version, which is actually simpler and doesn't require `True` nor `False`

Reply:
The purpose of my (flawed) logic was to disable the support bottom when using self support, and enable the support bottom when using soluble support.  I believe your change achieves the opposite, but perhaps I am reading it incorrectly.

Reply:
Paul suggested the extruder_nr == support_extruder_nr as a way to check if we are slicing for self support